src/main/java/org/ow2/proactive/connector/iaas/cloud/CloudManager.java

```diff
@@ -37,13 +38,29 @@
 import org.ow2.proactive.connector.iaas.model.Instance;
 import org.ow2.proactive.connector.iaas.model.InstanceScript;
 import org.ow2.proactive.connector.iaas.model.ScriptResult;
+import org.ow2.proactive.connector.iaas.model.Tag;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
+import lombok.Getter;
+
 
 @Service
 public class CloudManager {
 
+    @Getter
+    @Value("${connector-iaas-tag.key:proactive-connector-iaas}")
+    private String connectorIaasTagKey;
+
+    public static final String DEFAULT_CONNECTOR_IAAS_TAG_KEY = "proactive-connector-iaas";
```

Comment:
why do you need the default value if you can already specify the default value in the @Value("${connector-iaas-tag.key:proactive-connector-iaas}") ?